CI/windows/build_libopus.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+################################################################################
+# Windows libopus cross-compile build script
+################################################################################
+#
+# This script file can be included in build scripts for Windows or run directly
+#
+################################################################################
+
+# Halt on errors
+set -eE
+
+_build_product() {
+    cd "${PRODUCT_FOLDER}"
+
+    step "Configure ("${ARCH}")..."
+    git clean -dxf
+    ./autogen.sh
+    PKG_CONFIG_PATH="${BUILD_DIR}/lib/pkgconfig" \
+        LDFLAGS="-L${BUILD_DIR}/lib -static-libgcc" \
+        CPPFLAGS="-I${BUILD_DIR}/include -D_FORTIFY_SOURCE=0" \
+        ./configure \
+        --host=$WIN_CROSS_TOOL_PREFIX-w64-mingw32 \
```

Comment:
Use curlys (curls?): `${WIN_CROSS_TOOL_PREFIX}-w64-mingw32`

Reply:
This is probably safest?  It works right now though, so I might push this off till later.
```
"${WIN_CROSS_TOOL_PREFIX}-w64-mingw32"
```